src/content/docs/dropins/cart/cart-functions.mdx

```diff
@@ -204,10 +289,45 @@ getEstimatedTotals([
 ]);
 ```
 
-Here's the shape of the data returned from the `getEstimatedTotals` function:
+## getEstimateShipping
+
+The `getEstimateShipping` function returns a list of shipping methods and the estimated shipping costs, based on the country ID. The function calls the [`estimateShippingMethods`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#mutation-estimateShippingMethods) mutation. It also calls the [`countries`](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/countries/) query to populate the list of countries, the [`country`](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/country/) query to populate the list of regions within the selected country. and the [`storeConfig`](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/store-config/) query to help resolve the shipping costs.
 
 ```ts
-type EstimatedTotals = {
+export const getEstimateShipping = async (
+  address: EstimateAddressInput
+): Promise<any | null>
+```
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['address', 'EstimateAddressInput', 'No', 'Defines the shipping address for the estimate.'],
+  ]}
+/>
+
+The `EstimateAddressInput` object has the following parameters:
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['countryCode', 'string', 'Yes', "The two-letter code representing the user's country."],
+    ['postcode', 'string', 'No', "The user's ZIP or postal code."],
+    ['region', 'object', 'No', 'An object containing the region name, region code, and region ID.'],
+    ['region.region', 'string', 'No', 'The state or province name.'],
+    ['region.code', 'string', 'No', 'The address region code.'],
+    ['region.id', 'number', 'No', 'The unique ID for a pre-defined region.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that contains shipping data or null. The `EstimatedShipping` object has the following shape:
+
```

Comment:
**Suggestion:** Would be useful to mention that the event bus emits a `shipping/estimate` event containing data about the address and shipping method whenever this function is called.